src/utils/miscellaneous.js

```diff
@@ -280,19 +280,19 @@ export function getCookieByName(name) {
         // decode the cookie value
         // get all cookies
         document.cookie
-            // separate the string into an array of cookies
             .split(';')
+            // separate the string into an array of cookies
             // find the cookie by name
             .find(cookieStr => cookieStr.startsWith(`${name}=`))
-            // use only the value of the cookie
-            .slice(5)
+            ?.slice(5)
```

Comment:
```suggestion
            ?.slice(5) ?? ''
```
otherwise, 
```javascript 
cookieVal === decodeURIComponent(undefined) === 'undefined'
```
and the function will return `{undefined: ''}` instead of of `null`